wormchain/contracts/tools/__tests__/test_ntt_accountant.ts

```diff
@@ -104,17 +138,35 @@ const host = ci
   : devnetConsts.chains[3104].tendermintUrlLocal;
 // TODO: have a mnemonic dedicated for this test
 const mnemonic =
-  devnetConsts.chains[3104].accounts.wormchainNodeOfGuardian0.mnemonic;
+  "quality vacuum heart guard buzz spike sight swarm shove special gym robust assume sudden deposit grid alcohol choice devote leader tilt noodle tide penalty";
```

Comment:
Is this a different account than what is in the consts file?

Reply:
yes, only in that those accounts are on Terra while this one is on wormchain (added in this PR)